### PR TITLE
[feature/v1-053] Added new styles for Menu bar

### PR DESCRIPTION
**Pull request by NikitaKhadnevich**
At present this Pull request filled next changes:
1. Added _custom styles_ for selected elements;
2. Implemented function for _toggle clicked styles_;
3. _Update styles_ and created new;
4. Added _new types_ for work with this;

### CODE REVIEW

**Review by hauteballe — APPROVED** [review]

**Review by Julia-Maliutsina — DISMISSED** [review]

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/api/logOut/logOut.tsx [hunk 1]
```diff
@@ -5,14 +5,18 @@ import { apiClientWrapper } from 'api/base';
...
   const navigateTo = useNavigate();
   return useMutation(
-    async (initialData: string | unknown) => {
+    async () => {
       const apiClient = apiClientWrapper();
       try {
-        const logOutResponse = await apiClient.post(API.getToken, initialData);
+        const logOutResponse = await apiClient.post(API.getToken, {
+          login: 'user',
+          password: 'user',
```
[the remark this answers is not in the document]
Comment (NikitaKhadnevich) [reply frontend/src/api/logOut/logOut.tsx:18]: No there is only mocked data for init request

[the remark this answers is not in the document]
Comment (k-krasilnikova) [reply frontend/src/api/logOut/logOut.tsx:18]: Why we can't provide real data?

File: frontend/src/components/Layout/Menu/Menu.tsx [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+  return (
+    <MenuTabs>
+      <MenuTabsWrapper>
+        {menuList.map((item) => (
+          <TabWrapper
+            id={item.path}
+            key={item.title}
+            component={Link}
+            classes={
+              menuItem === item.path ? { root: classes.selected } : { root: classes.default }
```
[the remark this answers is not in the document]
Comment (NikitaKhadnevich) [reply frontend/src/components/Layout/Menu/Menu.tsx]: Resolved

File: frontend/src/api/logOut/logOut.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (NikitaKhadnevich) [reply frontend/src/api/logOut/logOut.tsx:18]: We will implemented it in 048 task

[the remark this answers is not in the document]
Comment (k-krasilnikova) [reply frontend/src/api/logOut/logOut.tsx:18]: Already merged, replace it please with endpoint from Eugene

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

**Review by k-krasilnikova — APPROVED** [review]

**Pull request merged**